spec/factories/service_requests.rb

```diff
@@ -1,4 +1,5 @@
 # Read about factories at https://github.com/thoughtbot/factory_girl
+require 'ffaker'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.